lib/run_loop/device_agent/client.rb

```diff
@@ -1464,6 +1464,14 @@ def launch_cbx_runner
           shutdown
         end
 
+        if device.simulator?
```

Comment:
We don't want to branch here.  `We should rely on CoreSimulator#launch` to detect whether a relaunch is necessary.

Reply:
Is `CoreSimulator` ignoring everything if the device is not sim?